extension/source/cells/mixtureHasChanged.ts

```diff
@@ -0,0 +1,77 @@
+export default function mixtureHasChanged<T>(
+  previous: T | undefined,
+  latest: T,
+): boolean {
+  return hasChanged(previous, latest);
```

Comment:
Could you use [_.isEqua](https://lodash.com/docs/4.17.15#isEqual) here instead on needing custom logic?

Reply:
Hmm that's tricky. I think we'd be fine right now but the problem is that cells care about internal mutability, and `Cell<Cell>` is a valid use-case. `mixtureHasChanged` provides a mechanism for this by using referential equality to distinguish two cells that currently have the same value but are referentially different.

Perhaps that's only a problem for `MemoryCell` 🤔. I have to think about this some more.

I'd prefer to keep it but I'm ok just using `_.isEqual` for now and revisiting this later if it feels off to you.

Reply:
It sounds like there may be some cases that `_.isEqual` or a similar function would not handle as we want, so I'm fine leaving it as is. In general, I like using external lib code for these kinds of things as they are more battle tested, handle edge cases we wouldn't think of, and reduce the code footprint we have to maintain.

Let's leave as is for this PR and consider revisiting in the future when/where it makes sense.